static/scss/components/whatsnew.scss

```diff
@@ -108,20 +109,30 @@ header .user-area:not(.arbitrary-class) .c-whatsnew-trigger {
                 border-style: none;
                 text-align: start;
 
+                img {
+                    width: $layout-xl;
+                }
+
                 h3 {
-                    @include text-body-sm;
+                    @include text-body-md;
+                    @include font-base;
                     font-weight: 500;
-                    padding-top: $spacing-sm;
+                    margin: 0;
+                    padding: $spacing-xs 0;
                 }
 
                 p {
                     @include text-body-sm;
+                    font-weight: 400;
+                    color: $color-dark-gray-05;
+                    padding-top: $spacing-xs;
```

Comment:
Heh, I realise I didn't do it myself when I added a padding to the `h3`, but in the spirit of having the distance between components being determined by their parent, setting a `gap` (and possibly its own padding) on `.c-whatsnew-entry-content` would be more robust.

No need to change it here since the use is so specific, but just wanted to call it out so we can form a habit of identifying these situations in the future :)